rotkehlchen/tests/utils/api.py

```diff
@@ -152,6 +180,9 @@ def wait_for_async_task(
                 )
             status = json_data['result']['status']
             if status == 'completed':
+                # Move status code to the outcome dict for easier checking
+                status_code = json_data['result'].get('status_code', HTTPStatus.OK)
```

Comment:
You shouldn't be introducing the default value. I would let it fail with key error if is not found. Otherwise you are modifying the response you are testing

Reply:
Hmm yeah, makes sense